### ISSUE TITLE
Unable to set permission when mounting additional partitions, misleading example

### ISSUE THREAD

**Issue by mengelmann** [andsens/bootstrap-vz#443]
It is not possible to set permission when mounting additional partitions using manifest file http://bootstrap-vz.readthedocs.io/en/master/manifests/index.html?highlight=mount#volume - mount point always has permission 0755. 

I'm not saying it is a high priority issue (we use single partition images), but it makes example manifest file https://github.com/andsens/bootstrap-vz/blob/master/manifests/examples/kvm/stretch-virtio-partitions.yml misleading - ```/tmp``` and ```/var/tmp``` should not have 0755 permissions but 1777.

**Comment by andsens** [comment]
The permissions are not meant for when you are bootstrapping, but for when the system actually boots. The options are added to the fstab:
https://github.com/andsens/bootstrap-vz/blob/19e15d15bfb3f886e80ff38dd7abd9976b261221/bootstrapvz/common/tasks/filesystem.py#L238

**Comment by mengelmann** [comment]
You can't set permission to the mount point via ```mount``` command or ```/etc/fstab``` file. The only valid solution is to mount && chmod && chown (if required) and then permissions will stay with the filesystem (at least with ext3, ext4). 

I encountered this issue while testing example manifests from bootstrap-vz repository.

**Comment by andsens** [comment]
My bad, I meant noexec & nosuid permissions. I don't see permission masks or owners mentioned anywhere in the docs or the manifest.

**Comment by mengelmann** [comment]
You're right, directory owner/permissions are not mentioned in docs nor the manifest.

I just noticed that example manifest https://github.com/andsens/bootstrap-vz/blob/master/manifests/examples/kvm/stretch-virtio-partitions.yml does not build a proper image - partitions for ```/tmp``` and ```/var/tmp``` have wrong permissions.

A simple way to correct these is using ```commands``` plugin. But they are mounted with improper permissions during the build, and it may influence the build (actually I got an error that it is impossible to create temporary apt repository GPG key during ```apt-get update```).
